django_object_actions/templates/django_object_actions/change_form.html

```diff
@@ -4,7 +4,7 @@
 {% block object-tools-items %}
   {% for tool in objectactions %}
     <li class="objectaction-item" data-tool-name="{{ tool.name }}">
-      <a href='tools/{{ tool.name }}/' title="{{ tool.standard_attrs.title }}"
+      <a href='{%url tools_view_name pk=object_id tool=tool.name %}' title="{{ tool.standard_attrs.title }}"
```

Comment:
whitespace consistency: I like keeping a single space between the braces so it's look like `{% url ... %}`